[modify/directory] - Add / Edit / Delete the previous files

Mavo39 opened the pull request:
- Move users folder under views folder the previous ones were under the layouts folder
- Delete the duplicates like donatioin-item show / delete modal / show-item / edit-item files
- Edit style to match other's layout
- Add users. on ProfileController to view the file
- Add 5 css files to avoid conflicts member will be able to have their own css file
- Add comment on style.css file to use common part like navigation bar

Mavo39 committed:
[modify/directory] - Add and Edit the previous files

- Move users folder under views folder
the previous ones were under the layouts folder
- Delete the duplicates like donatioin-item show / delete modal / show-item / edit-item files
- Edit style to match other's layout
- Add users. on ProfileController to view the file
- Add 5 css files to avoid conflicts
member will be able to have their own css file
- Add comment on style.css file to use common part like navigation bar

KobeSergio reviewed (changes requested):
Hello team i noticed that you guys opted to go with a seperated css approach, this is good so that conflicts are lessened out, but I suggest to rather having one .css per member, make it a practice to use a specific css file for the page itself when needed.
example:
I have a login page, the css that I use there is css/common.css+css/login.css combining the common css (what everyone is using) and the specific css (the css for that page only)
I have a profile page, the css that I use there is css/common.css+css/profile.css combining the common css (what everyone is using) and the specific css (the css for that page only)

Mavo39 committed:
[modify/directory] - Edit controller and Delete css files
Changes:

- Delete users. on HomeController
- Delete css files
- Move home.blade.php under the views folder

Mavo39 committed:
Merge branch 'main' into modify/directory

# Conflicts:
#	resources/views/home.blade.php
